config.js

```diff
@@ -27,10 +27,9 @@ module.exports = {
     host: process.env.PGHOST,
     port: process.env.PGPORT,
     database: process.env.PGNAME || process.env.DB_NAME || NAME,
-    user: process.env.PGUSER,
-    password: process.env.PGPASS,
+    user: process.env.PGUSER || 'postgres',
```

Comment:
This will break the defaults for OSX - but I see that the instructions suggest `printf "PGUSER=$USER" > .env` so :+1: but heads up to the whole team that this will change the default.